test/checkpoint/test_checkpoint.c

```diff
@@ -373,6 +373,9 @@ enable_disagg(const char *mode)
         fprintf(stderr, "Invalid disaggregated mode: %s\n", mode);
         return EINVAL;
     }
+
+    g.opts.palm_map_size_mb = 2048; /* Set 2GB map size for palm by default. */
+
```

Comment:
we dont need 
`pts.build_dir = (char *)BUILDDIR;`?

Reply:
It is already set by testutil. No need to set explicitly.